client/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "stage": 0
```

Comment:
We're using the full power of Babel and ES7. Note, we're not moving to Babel 6 yet.